changelog.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina RabbitMQ package through the releases.
```

Comment:
Follow this

https://github.com/ballerina-platform/module-ballerina-websocket/pull/113/files#diff-3bd14d078188074c410028847113ceae68865d0ad5b844a27183ef87fbe2fcc3

Reply:
Fixed in ca9c9e65dcd85a834e5fbb7f51d7d6aef3f82936. And nothing to mention in other sections (i.e Added, Removed, Fixed etc)